crates/autopilot/src/domain/settlement/trade.rs

```diff
@@ -0,0 +1,46 @@
+use crate::domain::{self, auction::order, eth};
+
+#[derive(Debug)]
+#[allow(dead_code)]
+pub struct Trade {
+    order_uid: domain::OrderUid, // todo order::Uid
```

Comment:
Is this an actual todo? If so can you add some more context to the comment on what is needed to make this happen (and ideally link an issue) or otherwise remove the commen?

Reply:
Not actually needed. Just a naming thing. Will remove it and do it in a separate PR.